perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue

```diff
@@ -27,32 +27,61 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
           :height="carousselHeight"
           hide-delimiters
           cycle
-          class="carousselParent clickable"
-          @click="openProductDetail">
+          class="carousselParent">
           <template v-if="product.imageFiles">
             <v-carousel-item
               v-for="(imageFile,i) in product.imageFiles"
               :key="i"
               :src="imageFile.src"
               max="300"
-              class="carousselImage" />
-            <v-btn
-              absolute
-              color="white"
-              class="productCardPrice"
-              fab
-              left
-              small
-              top>
-              {{ symbol }} {{ product.price }}
-            </v-btn>
+              class="carousselImage">
+              <v-toolbar
+                color="transparent"
+                class="toolbarCard"
+                flat>
+                <v-btn
+                  absolute
+                  color="white"
+                  class="productCardPrice"
+                  fab
+                  small
+                  top>
+                  {{ symbol }} {{ product.price }}
+                </v-btn>
+                <v-spacer />
+                <v-menu
+                  v-if="userData.canEdit"
+                  v-model="showMenu"
+                  offset-y
+                  attach
+                  :nudge-left="110">
+                  <template v-slot:activator="{ on}">
+                    <v-btn
+                      dark
+                      :title="$t('exoplatform.perkstore.button.editProduct', {0: product.title})"
+                      icon
+                      class="productCardAction mr-0"
+                      @blur="closeMenu"
+                      v-on="on">
+                      <v-icon>mdi-dots-vertical</v-icon>
+                    </v-btn>
+                  </template>
+                  <v-list class="pa-0 ma-0">
+                    <v-list-item @mousedown="$event.preventDefault()">
+                      <v-list-item-title class="editProductMenu" @click="$emit('edit', product)">
+                        <i class="uiIconEdit"> </i>{{ $t('exoplatform.perkstore.button.menuEditProduct') }}
+                      </v-list-item-title>
+                    </v-list-item>
+                  </v-list>
+                </v-menu>
+              </v-toolbar>
+            </v-carousel-item>
           </template>
           <v-expand-transition>
```

Comment:
the content of v-expand-transition will be deleted and added in the CardFlip
you have to take this into consideration